subjects/printandmiss/README.md

```diff
@@ -0,0 +1,46 @@
+## printandmiss
+
+### Instructions
+
+Write a function called `PrintAndMiss` that takes a string and an integer and return a string containing the characters until reaching the integer, then skipping that same number of characters, and repeats until the end of the string.
+
+- If the string is empty return `Invalid Output` followed by newline `\n`.
+- If the integer is `0` or it's negative return the string followed by a newline `\n`.
+
+
+### Expected function
+
+```go
+func PrintAndMiss(arg string, loop int) string {
```

Comment:
change the name of the second param to num like the test

Reply:
Done